src/index/base.cpp

```diff
@@ -18,8 +18,8 @@ using node::ReadBlockFromDisk;
 
 constexpr uint8_t DB_BEST_BLOCK{'B'};
 
-constexpr int64_t SYNC_LOG_INTERVAL = 30; // seconds
-constexpr int64_t SYNC_LOCATOR_WRITE_INTERVAL = 30; // seconds
+constexpr std::chrono::steady_clock::duration SYNC_LOG_INTERVAL{30s};
+constexpr std::chrono::steady_clock::duration SYNC_LOCATOR_WRITE_INTERVAL{30s};
```

Comment:
This can stay `auto`

Reply:
Thanks. Done in https://github.com/bitcoin/bitcoin/pull/25079/commits/0374ee8ea0e16f428f6f391fe5cf92446e3f8c5a